applications/GeoMechanicsApplication/custom_constitutive/truss_backbone_constitutive_law.cpp

```diff
@@ -0,0 +1,166 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+// Project includes
+#include "truss_backbone_constitutive_law.h"
+#include "geo_mechanics_application_variables.h"
+#include "includes/properties.h"
+
+namespace Kratos
+{
+ConstitutiveLaw::Pointer TrussBackboneConstitutiveLaw::Clone() const
+{
+    return Kratos::make_shared<TrussBackboneConstitutiveLaw>(*this);
+}
+
+void TrussBackboneConstitutiveLaw::GetLawFeatures(Features& rFeatures)
+{
+    // Set the strain size
+    rFeatures.mStrainSize = 1;
+
+    // Set the space dimension
+    rFeatures.mSpaceDimension = 3;
```

Comment:
In my opinion, these comments repeat what the code already expresses clearly:
```suggestion
    rFeatures.mStrainSize = 1;
    rFeatures.mSpaceDimension = 3;
```

Reply:
Done.